launch_ros/launch_ros/actions/node.py

```diff
@@ -257,21 +256,22 @@ def get_nested_dictionary_from_nested_key_value_pairs(params):
         normalized_params = []
         for param in params:
             from_attr = param.get_attr('from', optional=True)
-            allow_substs = param.get_attr('allow_substs', optional=True)
-            name = param.get_attr('name', data_type=bool, optional=True)
+            allow_substs = param.get_attr('allow_substs', data_type=bool, optional=True)
```

Comment:
@ivanpauno now that `allow_substs` can be a `Substitution`, should it be handled differently here?

Reply:
Fixed in https://github.com/ros2/launch_ros/pull/168/commits/3e3da2f17c6a90db7d1e2957878dd6f50bc2da93.